internal/pkg/dynamo/dao/merchant/ddb_tiered_merchant_offer_table.go

```diff
@@ -0,0 +1,43 @@
+package merchant
+
+import(
+	"github.com/blucard/blucard/internal/model/merchant"
+)
+
+// Schema for a table with a single entry containing a map from tier to delegate table name
+const tieredMerchantOfferTableSchema = struct {
+	tableName string
```

Comment:
I don't think it's necessary to pass in the table name for the dao. this can probably be initialized as a `const`. if we decided to have different tables for environments, we can pass the environment instead.

the previous dao makes sense since it's going to be essentially copies of the same dao with different names, but this is probably only ever going to have 1 instance